pkg/provider-local/webhook/istio/add.go

```diff
@@ -60,7 +61,7 @@ func AddToManagerWithOptions(mgr manager.Manager, _ AddOptions) (*extensionswebh
 		Target:            extensionswebhook.TargetSeed,
 		Path:              WebhookName,
 		Webhook:           &admission.Webhook{Handler: handler, RecoverPanic: ptr.To(true)},
-		NamespaceSelector: &metav1.LabelSelector{MatchLabels: map[string]string{v1beta1constants.GardenRole: v1beta1constants.GardenRoleIstioIngress}},
+		NamespaceSelector: &metav1.LabelSelector{MatchLabels: map[string]string{corev1.LabelMetadataName: "istio-ingress"}},
```

Comment:
This selector does not cover the `virtual-garden-istio-ingress` namespace. The e2e test discovered this immediately ([ref](https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_gardener/12345/pull-gardener-e2e-kind-ha-single-zone/1937894068918947840#1:build-log.txt%3A555)) 😄